fix crash in message store

":["Conversation","Conversation_gaspos"],"Props":{"appSource":"hubspot","channel":"EMAIL","id":"4cd9637b-d272-4619-8f5f-8126da57af7f","initiatorFirstName":"Joshua","initiatorLastName":"Smith","initiatorType":"USER","initiatorUsername":"josh.smith@gaspos.co","messageCount":2,"source":"hubspot","sourceOfTruth":"hubspot","startedAt":"2018-03-02T14:04:05.449Z","status":"ACTIVE","syncDate":"2023-02-01T12:22:03.939836552Z","threadId":"0a8eeaff3f60e8871186706b974eb36d","updatedAt":"2018-03-02T14:10:56.207Z"}}]
2023-03-01 13:33:54.304   BOLT  [bolt-25685@neo4j-core-e38fe2eb-35.production-orch-0010.neo4j.io:7687] S: RECORD [{"Id":182591,"ElementId":"4:9911eaca-fdc0-4faa-9ecc-9bf88249b7de:182591","Labels":["Conversation","Conversation_gaspos"],"Props":{"appSource":"hubspot","channel":"EMAIL","id":"7414046c-e953-4823-903f-a6f73b054c19","initiatorFirstName":"Joshua","initiatorLastName":"Smith","initiatorType":"USER","initiatorUsername":"josh.smith@gaspos.co","messageCount":1,"source":"hubspot","sourceOfTruth":"hubspot","startedAt":"2018-03-02T13:13:27.864Z","status":"ACTIVE","syncDate":"2023-02-01T12:22:03.939836552Z","threadId":"db8bc452a0930f9effb872e10346c25f","updatedAt":"2018-03-02T13:13:27.864Z"}}]
2023-03-01 13:33:54.304   BOLT  [bolt-25685@neo4j-core-e38fe2eb-35.production-orch-0010.neo4j.io:7687] S: SUCCESS {"t_first":"-1","t_last":"5","db":"neo4j"}
2023-03-01 13:33:54.304   BOLT  [bolt-25685@neo4j-core-e38fe2eb-35.production-orch-0010.neo4j.io:7687] C: COMMIT
2023-03-01 13:33:54.318   BOLT  [bolt-25685@neo4j-core-e38fe2eb-35.production-orch-0010.neo4j.io:7687] S: SUCCESS {"t_first":"-1","bookmark":"FB:kcwQmRHqyv3AT6qezJv4gkm33soABi84kA==","t_last":"-1"}
panic: interface conversion: interface {} is nil, not string

goroutine 27093 [running]:
github.com/openline-ai/openline-customer-os/packages/server/message-store-api/service.mapNodeToConversation(0xc00040f628)
        /workspace/build/message-store-api/service/CustomerOSService.go:559 +0x994
github.com/openline-ai/openline-customer-os/packages/server/message-store-api/service.(*CustomerOSService).GetConversations(0xfa68a0?, {0xfa68a0?, 0xc0007ba3f0}, {0xc00051116a, 0x6}, 0x0)
        /workspace/build/message-store-api/service/CustomerOSService.go:350 +0x435
github.com/openline-ai/openline-customer-os/packages/server/message-store-api/service.(*MessageService).GetFeeds(0xc000596440, {0xfa68a0, 0xc0007ba3f0}, 0xc0000ac460)
        /workspace/build/message-store-api/service/MessageStoreService.go:155 +0xfb
github.com/openline-ai/openline-customer-os/packages/server/message-store-api/proto/generated._MessageStoreService_GetFeeds_Handler({0xdfc4c0?, 0xc000596440}, {0xfa68a0, 0xc0007ba3f0}, 0xc0004a4230, 0x0)

--- a/packages/server/message-store-api/service/CustomerOSService.go
+++ b/packages/server/message-store-api/service/CustomerOSService.go
@@ -551,15 +551,33 @@ func mapNodeToConversation(node *dbtype.Node) *Conversation {
 	props := utils.GetPropsFromNode(*node)
 
 	conversation := new(Conversation)
-	conversation.Id = utils.GetPropsFromNode(*node)["id"].(string)
-	conversation.Status = utils.GetPropsFromNode(*node)["status"].(string)
-	conversation.Channel = utils.GetPropsFromNode(*node)["channel"].(string)
-	conversation.StartedAt = utils.GetPropsFromNode(*node)["startedAt"].(time.Time)
-	conversation.UpdatedAt = utils.GetPropsFromNode(*node)["updatedAt"].(time.Time)
-	conversation.InitiatorFirstName = utils.GetPropsFromNode(*node)["initiatorFirstName"].(string)
-	conversation.InitiatorLastName = utils.GetPropsFromNode(*node)["initiatorLastName"].(string)
-	conversation.InitiatorUsername = utils.GetPropsFromNode(*node)["initiatorUsername"].(string)
-	conversation.InitiatorType = utils.GetPropsFromNode(*node)["initiatorType"].(string)
+	if id, err := utils.GetPropsFromNode(*node)["id"].(string); err {
+		conversation.Id = id
+	}
+	if status, err := utils.GetPropsFromNode(*node)["status"].(string); err {
+		conversation.Status = status
+	}
+	if channel, err := utils.GetPropsFromNode(*node)["channel"].(string); err {
+		conversation.Channel = channel
+	}
+	if channel, err := utils.GetPropsFromNode(*node)["startedAt"].(time.Time); err {
+		conversation.StartedAt = channel
+	}
+	if updatedAt, err := utils.GetPropsFromNode(*node)["updatedAt"].(time.Time); err {
+		conversation.UpdatedAt = updatedAt
+	}
+	if initatorFirstName, err := utils.GetPropsFromNode(*node)["initiatorFirstName"].(string); err {
+		conversation.InitiatorFirstName = initatorFirstName
+	}
+	if initatorLastName, err := utils.GetPropsFromNode(*node)["initiatorLastName"].(string); err {
+		conversation.InitiatorLastName = initatorLastName
+	}
+	if initatorUsername, err := utils.GetPropsFromNode(*node)["initiatorUsername"].(string); err {
+		conversation.InitiatorUsername = initatorUsername
+	}
+	if initatorType, err := utils.GetPropsFromNode(*node)["initiatorType"].(string); err {
+		conversation.InitiatorType = initatorType
+	}
 
 	conversation.LastSenderId = utils.GetStringPropOrEmpty(props, "lastSenderId")
 	conversation.LastSenderType = utils.GetStringPropOrEmpty(props, "lastSenderType")

